pkg/controller/rest/vdri/operation.go

```diff
@@ -81,6 +91,49 @@ func (o *Operation) CreatePublicDID(rw http.ResponseWriter, req *http.Request) {
 	rest.Execute(o.command.CreatePublicDID, rw, bytes.NewReader(reqBytes))
 }
 
+// SaveDID swagger:route POST /vdri/did vdri saveDIDReq
+//
+// Saves a did document with the friendly name.
+//
+// Responses:
+//    default: genericError
+//        200: emptyRes
```

Comment:
no need of empty response, swagger takes care of it if none specified

Reply:
done